common/convert/convert.go

```diff
@@ -77,3 +78,22 @@ func UnixMillis(t time.Time) int64 {
 func RecvWindow(d time.Duration) int64 {
 	return int64(d) / int64(time.Millisecond)
 }
+
+// SplitFloatDecimals takes in a float64 and splits
+// the decimals into their own integers
+func SplitFloatDecimals(input float64) (baseNum, decimalNum int64, err error) {
+	decStr := strconv.FormatFloat(input, 'f', -1, 64)
+	splitNum := strings.Split(decStr, ".")
+	baseNum, err = strconv.ParseInt(splitNum[0], 10, 64)
+	if err != nil {
+		return 0, 0, err
+	}
+	if len(splitNum) == 1 {
```

Comment:
can u pls move check len(decStr) != 1 before strings.split so incase if it is 0, its more efficient because then we wont have to do strings.Split